audio/libopenmpt/Portfile

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           legacysupport 1.1
+
+# Undefined symbols for architecture x86_64:
+# "typeinfo for std::bad_variant_access"
+legacysupport.use_mp_libcxx yes
```

Comment:
```suggestion
legacysupport.newest_darwin_requires_legacy 12
legacysupport.use_mp_libcxx yes
```

Reply:
@aeiouaeiouaeiouaeiouaeiouaeiou Isn’t this already determined by the base somewhere?

P. S. It is in fact everything prior to Sierra, it seems: https://ports.macports.org/port/libopenmpt/details

Reply:
I'm wondering why Mojave is broken as well.

Reply:
Cannot test that and logs are dead, but perhaps for unrelated reason?